daru.gemspec

```diff
@@ -74,6 +74,8 @@ EOF
   spec.add_development_dependency 'ruby-prof'
   spec.add_development_dependency 'simplecov'
   spec.add_development_dependency 'nokogiri'
-  spec.add_development_dependency 'guard-rspec'
   spec.add_development_dependency 'gruff'
+  if RUBY_VERSION >= '2.2.5'
```

Comment:
I get its reference from [here](https://www.tiredpixel.com/2014/01/05/curses-conditional-ruby-gem-installation-within-a-gemspec/) (See at the end). And its working.